src/main/resources/features/moth/54 - Print MOT certificate.feature

```diff
@@ -0,0 +1,23 @@
+@mothprint @mothint
+Feature: 54 - Check whether MOT certificates can be downloaded after entering the V5C number
+
+  Scenario: A MOTH user searches for a vehicle with an MOT test and downloads the MOT certificate
+    Given I browse to /
+    And I enter "CGSENOK" in the registration field
+    When I press the "Continue" button
+
+    And I click the accordion section with the id "mot-history-description"
+    And I click the first "View test certificate" text
+    And The page contains "Enter latest V5C number"
```

Comment:
Scenarios should follow the Given/When/Then pattern - Its hard to understand here what is the path taken and what is the expected result here

Reply:
Done